homework.cls

```diff
@@ -119,7 +138,7 @@
 % Miscellaneous %
 %%%%%%%%%%%%%%%%%
 
-\RequirePackage{lmodern}            % Use the Latin Modern font
+\RequirePackage{mathpazo}           % Use the Palatino font
```

Comment:
There used to be the option to customize the font when loading the class. For some reason I removed it and a lot of other options in 4cca36dc4a7ecef753db867b04f6cf525e9c2e2e back in 2017 without explaining myself 😅. At any rate, I understand liking Palatino as the font, but I prefer Latin Modern, so I'd like this to be user-configurable instead of enforced

Reply:
Looking back, it was probably removed because there were too many hard-coded font options

https://github.com/artemmavrin/latex-homework/blob/68b423ff99e381a4bd091fab4dbb15f2960bbae5/homework.cls#L23-L32

https://github.com/artemmavrin/latex-homework/blob/68b423ff99e381a4bd091fab4dbb15f2960bbae5/homework.cls#L164-L224